crate-wasm/src/lib.rs

```diff
@@ -10,38 +10,24 @@ use std::f64::consts;
 #[global_allocator]
 static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
 
-fn in_cardioid(x: f64, y: f64) -> bool {
-    let a = x - 1.0 / 4.0;
-    let q = a * a + y * y;
-
-    q * (q + a) <= 0.25 * y * y
-}
-
-fn in_bulb(x: f64, y: f64) -> bool {
-    let a = x + 1.0;
-
-    a * a + y * y <= 0.0625 // <= 1/16
-}
-
 // how many iterations does it take to escape?
 fn get_escape_time(
     x: f64,
     y: f64,
     max_iterations: u32,
     escape_radius: f64,
     is_smoothed: bool,
+    exponent: u32
 ) -> (u32, f64) {
-    if in_cardioid(x, y) || in_bulb(x, y) {
-        return (max_iterations, 0.0);
-    }
-
     let c: Complex64 = Complex64::new(x, y);
     let mut z: Complex64 = c.clone();
+    let e: Complex64 = Complex64::new(exponent.into(),0.0);
+
 
     let mut iter: u32 = 0;
     while z.norm() < escape_radius && iter < max_iterations {
         iter += 1;
-        z = z * z + c;
+        z = z.powc(e) + c;
```

Comment:
Here you convert `exponent` to Complex64 and use `powc`. Would keeping it as an unsigned int and using `powu` instead also work? I think it would be more efficient